### ISSUE TITLE
Migliorare copia/incolla codice

### ISSUE THREAD

**Issue by afrittella** [laravel-italia/site#7]
Provando a copiare e ad incollare il codice presente su qualsiasi articolo o tutorial, il codice viene incollato senza formattazione e con il link alla pagina copiata, con l'aggiunta del tag di ShareThis.

Provato con Google Chrome e Mozilla Firefox su Sistema operativo Mac OS 10.11.5

Es. 
[Creare un Blog con Laravel - Inserire dei dati di prova...](http://laravel-italia.it/articoli/creare-un-blog-con-laravel-5/parte-5-inserire-dei-dati-di-prova-con-il-seeding)

Copiando il primo box di codice si ottiene questo risultato:

<?php use Illuminate\Database\Seeder; use Illuminate\Database\Eloquent\Model; class DatabaseSeeder extends Seeder { /*\* \* Run the database seeds. \* \* @return void */ public function run() { Model::unguard(); // $this->call('UserTableSeeder'); } } - See more at: http://laravel-italia.it/articoli/creare-un-blog-con-laravel-5/parte-5-inserire-dei-dati-di-prova-con-il-seeding#sthash.b1r9bq10.dpuf

**Comment by ChristianGiupponi** [comment]
Concordo con @afrittella, in questo modo rendiamo troppo complicato per un utente seguire i tutorial.
